src/views/repository/Trainings.vue

```diff
@@ -5,11 +5,26 @@
     :error-code="errorCode">
     <div
       v-if="repository"
-      class="trainings">
-      <div class="trainings__new-example">
-        <new-example-form :repository="repository" />
+      class="repository-trainings">
+      <div class="repository-trainings__new-example">
+        <div
+          v-if="repository.authorization &&
```

Comment:
Check if user is not authenticated, after check if can cantribute in this Repository.